Docs/SupportedAPIs.md

```diff
@@ -69,3 +69,55 @@ Contributions to expand support to unimplemented functionality are always welcom
 | POST   | `/session/:sessionId/window/:windowHandle/maximize` | Supported    | Not implemented     |
 | GET    | `/session/:sessionId/window_handle`                 | Supported    | Not implemented     |
 | GET    | `/session/:sessionId/window_handles`                | Supported    | Not implemented     |
+
+
+## Example Usage
```

Comment:
This is cool, but it seems buried here.  I would actually move all of this to it's own file, `Docs/Usage.md` or something like that.  That way it'll be more discoverable!

Reply:
That's a good idea I'll push up a commit with that change.